.swiftlint.yml

```diff
@@ -117,8 +117,8 @@ only_rules:
   - force_try
   # Force unwrapping should be avoided.
   - force_unwrapping
-  # Prefer to locate parameters with defaults toward the end of the parameter list.
-  - function_default_parameter_at_end
+  # # Prefer to locate parameters with defaults toward the end of the parameter list.
+  # - function_default_parameter_at_end
```

Comment:
We can completely remove it if we want to.

Reply:
done; thank you!